tools/cibadmin.c

```diff
@@ -313,19 +317,19 @@ main(int argc, char **argv)
                 crm_log_args(argc, argv);
                 break;
             case 'a':
-                output = createEmptyCib(1);
-                if (optind < argc) {
-                    crm_xml_add(output, XML_ATTR_VALIDATION, argv[optind]);
-                }
-                admin_input_xml = dump_xml_formatted(output);
-                fprintf(stdout, "%s\n", crm_str(admin_input_xml));
-                crm_exit(CRM_EX_OK);
+                cib_action = cib_action_EMPTYCIB;
                 break;
             default:
                 printf("Argument code 0%o (%c)" " is not (?yet?) supported\n", flag, flag);
                 ++argerr;
                 break;
         }
+        if (cib_action_old != NULL && cib_action != cib_action_old) {
+            printf("Combining multiple commands into one go not supported\n");
+            ++argerr;
+	    break;
+        }
```

Comment:
How does this prevent specifying such aliases?  If course, if
someone unreasonably relied upon undefined behaviour, they are going,
deservedly, to be bitten.  But nothing unexpected, you simply don't
do stupid things like that.

Definitely better then allowing proliferation of ill-advised command
combinations.  Do you know of any command that would allow combining
action triggers like that, relying on some undocumented decision rule
about what to actually trigger?  It's simply unsupportable and better
to indicate that clearly, to prevent bad surprises (that it wasn't
the case in the past is already bad enough).

Reply:
It also prevents command injection should any of the official or
home-made higher level management tools be naive enough, so you can
take it also a very desired security hardening -- you don't want
anyone be able to completely shift the semantics of the command
to execute like that.

Reply:
Yes, exactly because it's not a good idea to surprise users,
it's better not to bet they will never forget extra dash as described.
I don't see a value in extra complexity only to be removed one sweep
later, do you?

Reply:
Especially when higher level tools that may perhaps call down to this
are somewhat expected to construct every such command in a sane
way, meaning they shall not be influenced a single bit.

Verging on the undefined behaviour is always unwise.

That works both ways, we are not doing a good job of positively
defining what exactly is this or other option/switch/subcommand
meant to trigger (see recent discussions, `--text-fancy`, what
are stability-over-time expectations -- they are actually more
important than whether a mix of actions, something bad from
beginning, is recognized deterministically over time).